src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -52,4 +52,7 @@ public net.minecraft.client.multiplayer.ClientAdvancementManager field_192803_d
 
 public net.minecraft.block.DoorBlock func_196426_b(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Z)V # DoorBlock playsound
 
-public net.minecraft.entity.Entity field_213326_aJ # eyeHeight
\ No newline at end of file
+public net.minecraft.entity.Entity field_213326_aJ # eyeHeight
+
+# vbo draw
```

Comment:
Why would you need this at here?

Reply:
I could probably just copy that function but I think using the vanilla is safer cuz of vanilla change
usage: uploading vbo repeatedly without using bufferbuilder every frame